index.html

```diff
@@ -10,7 +10,145 @@
 </head>
 
 <body>
-    <h1>Portfolio Project</h1>
+    <header>
+        <svg width="75" height="18" viewBox="0 0 75 18" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
- [ ] As the last code reviewer said 👇🏽 

> Kindly make the logo of the page a link by using the a tag, this way when the user clicks on it, it takes him to the top of the page 👍

> [Optional]

>  I think it's better if you just write it as a text, like this <a href="#">My Logo<a>. It's shorter and less messy than the SVG

- [ ] Please 🙏🏽 complete all the previous requirements before asking for a new code review request, because you have limited chances to approve the project.